sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -191,6 +204,11 @@ export class BatchingReceiver extends MessageReceiver {
           clearTimeout(totalWaitTimer);
         }
 
+        if (cleanupAbortSignalFn) {
+          cleanupAbortSignalFn();
```

Comment:
Doesn't `cleanupAbortSignalFn` currently reject the promise? Won't that cause us to reject even if we have messages that we intend to resolve with?

Reply:
I can rename that function. It doesn't cause the abort handler to fire. It actually does the opposite - it removes the abort signal handlers since we're done.

Reply:
Oh I see, if you called it with an error, then it would reject, is that right?

Reply:
Will be changed to `removeAbortSignalListenersFn`

Reply:
Changed.